docs/integrations/saas-cloud/box.md

```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
+   </td>
+   <td>The Source will retry for up to 90 minutes, after which it quits.
+   </td>
+   <td>ThirdPartyGenericError
+   </td>
+  </tr>
+  <tr>
+   <td>FirstPartyGeneric
+   </td>
+   <td>Normally due to an error communicating with the internal Sumo Logic APIs.
+   </td>
+   <td>Yes
+   </td>
+   <td>The Source will retry for up to 90 minutes, after which it quits.
+   </td>
+   <td>FirstPartyGenericError
+   </td>
+  </tr>
+</table>
+
+
+
+#### JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the [Collector Management API](/docs/api/collectors). See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details.
+
+<table>
+  <tr>
+   <td><strong>Parameter</strong>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Required?</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Access</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>config
+   </td>
+   <td>JSON Object
+   </td>
+   <td>Yes
+   </td>
+   <td>Contains the<a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/Duo-Source#configParameters"> configuration parameters</a> for the Source.
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>schemaRef
+   </td>
+   <td>JSON Object
+   </td>
+   <td>Yes
+   </td>
+   <td>Use <code>&#123;"type":"Box"&#125;</code> for a Box Source.
+   </td>
+   <td>not modifiable
+   </td>
+  </tr>
+  <tr>
+   <td>sourceType
+   </td>
+   <td>String
+   </td>
+   <td>Yes
+   </td>
+   <td>Use <code>Universal</code> for a Box Source.
+   </td>
+   <td>not modifiable
+   </td>
+  </tr>
+</table>
+
+
+The following table shows the **config** parameters for a Box Source.
+
+<table>
+  <tr>
+   <td><strong>Parameter</strong>
+   </td>
+   <td><strong>Type</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Default</strong>
+   </td>
+   <td><strong>Description</strong>
+   </td>
+   <td><strong>Access</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>name
+   </td>
+   <td>String
+   </td>
+   <td>Yes
+   </td>
+   <td>
+   </td>
+   <td>Type a desired name of the Source. The name must be unique per Collector. This value is assigned to the <a href="/docs/search/Get-Started-with-Search/Search-Basics/Built-in-Metadata">metadata</a> field <code>_source</code>.
+   </td>
+   <td>modifiable
+   </td>
+  </tr>
+  <tr>
+   <td>description
+   </td>
+   <td>String
+   </td>
+   <td>No
+   </td>
+   <td>null
+   </td>
+   <td>Type a description of the Source.
+   </td>
+   <td>modifiable
+   </td>
+  </tr>
+  <tr>
+   <td>category
+   </td>
+   <td>String
+   </td>
+   <td>No
+   </td>
+   <td>null
+   </td>
+   <td>Type a category of the source. This value is assigned to the <a href="/docs/search/Get-Started-with-Search/Search-Basics/Built-in-Metadata">metadata</a> field <code>_sourceCategory</code>. See <a href="/docs/send-data/best-practices#good-and-bad-source-categories">best practices</a> for details.
```

Comment:
<td>Type a category for the source. This value is assigned to the <a href="/docs/search/Get-Started-with-Search/Search-Basics/Built-in-Metadata">metadata</a> field <code>_sourceCategory</code>.

Reply:
What's the change here?

Reply:
<img width="1374" alt="Screen Shot 1401-08-30 at 22 57 26" src="https://user-images.githubusercontent.com/116072777/203121513-461599e9-9621-4fd4-ba8e-531af786df4c.png">

Reply:
It looks like her change was:

Type a category _**for**_ the source.

Reply:
@kimsauce: Thanks for helping me to understand the change. 
@stacykor: I believe "of" is more appropriate here as it says about "some connection to object" here (category and source). We use "for" to show reasons or purpose. Let me know if we can agree on this and close it.

Reply:
No. You need to look at how you're using the definite article. 
"Type the category of the source".  Acceptable 
"Type a category for the source." Which category does the source belong to. 

Pick one but you can't say Type a category of the source. It's its indefinite.

Reply:
The category is not for the source, it is "of" the source. 
The first one is acceptable, however, we should avoid using 2 "the" within the same sentence in such a short phrase. For now, I am keeping the first one.